skypy/pipeline/tests/test_skypy.py

```diff
@@ -36,36 +35,39 @@ def test_skypy():
     assert e.value.code == 2
 
     # Process empty config file
-    filename = get_pkg_data_filename('data/empty_config.yml')
-    assert skypy.main([filename, 'empty.fits']) == 0
+    config_filename = get_pkg_data_filename('data/empty_config.yml')
```

Comment:
why don't we use a temporary YAML file here too?

Reply:
This is a (non-temporary) input file that is part of the repository